README.md

```diff
@@ -144,6 +144,8 @@ $ xxd -g 2 -l 64 -p /dev/random | tr -d '\n'
 * `IMGPROXY_CONCURRENCY` — the maximum number of image requests to be processed simultaneously. Default: double number of CPU cores;
 * `IMGPROXY_MAX_CLIENTS` — the maximum number of simultaneous active connections. Default: `IMGPROXY_CONCURRENCY * 5`;
 * `IMGPROXY_TTL` — duration in seconds sent in `Expires` and `Cache-Control: max-age` headers. Default: `3600` (1 hour);
+* `IMGPROXY_CACHE_FILES` — boolean value, sets whether a response sould be cached or not. If `false`, then ImgProxy will always return the status 200 and an
```

Comment:
I don't like the name. In fact, we don't do caching at all. `IMGPROXY_USE_ETAG` would be much better.